src/electron/views/results/test-view.tsx

```diff
@@ -60,6 +60,7 @@ export const TestView = NamedFC<TestViewProps>('TestView', props => {
                 scanMetadata={scanMetadata}
                 shouldAlertFailuresCount={true}
                 cardsViewData={cardsViewData}
+                showTabStops={true}
```

Comment:
Decided to leave as is here (since the property is not used/won't impact users). In a future PR, this will be passed from store data.

Reply:
Actually on second thought, I'll leave this out.